docs/octopus-rest-api/octopus-cli/allow-releaseprogression.md

```diff
@@ -6,13 +6,13 @@ description: Allows a release to progress to next phase
 The [Octopus CLI](/docs/octopus-rest-api/octopus-cli/index.md) can be used to allow a release to progress to next phase.
 
 ```text
-Allows a release to progress to next phase.
+Allow a release from progressing to next phase.
```

Comment:
@andyinaus I'm struggling to parse this sentence. I think some alternatives are:
Allow a release to progress to the next phase.
Allow a release to move from progressing to the next phase.
Is that something you can change in the CLI tool?

Reply:
It's coming from the CLI command description, which I just updated previously but having some issues with the release, and I haven't got time to address the release.